probes/utils/fuzzing.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"embed"
+	"fmt"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/finding"
+)
+
+func FuzzerRun(raw *checker.RawResults, fs embed.FS, probeID, fuzzerName string) ([]finding.Finding, string, error) {
```

Comment:
so what I'm noticing is that every check you convert has some shared functionality it needs, but because each probe is in a different package, you've been sticking it in this `utils` folder.

What do you think of adding some sub-folders to organize the functionality? And if you do it under an `internal` path, then no one can depend on it? This would also avoid the "util" package name

```
"github.com/ossf/scorecard/v4/probes/internal/fuzzing"
"github.com/ossf/scorecard/v4/probes/internal/tools"
```

or if you want to keep the "util" folder:
```
"github.com/ossf/scorecard/v4/probes/internal/util/fuzzing"
"github.com/ossf/scorecard/v4/probes/internal/util/tools"
```

Reply:
done.